learn-arai/app/hooks/useAuth.ts

```diff
@@ -0,0 +1,80 @@
+'use client';
+
+// this react hook responsible for checking if user is logged in.
+import { useEffect, useState } from 'react';
+
+import { useLocalStorage } from './useLocalStorage';
+import { useUser } from './useUser';
+
+export const useAuth = () => {
+    const { addUser, removeUser, user } = useUser();
+    const { getItem } = useLocalStorage();
+    const [isAuthenticated, setIsAuthenticated] = useState<boolean>();
```

Comment:
You don't need this anymore if user isn't authenticated, just `removeUser()` which set user as null

Reply:
`[isAuthenticated, setIsAuthenticated]` is deleted.

Reply:
It's still not deleted